{{cookiecutter.project_slug}}/config/settings/base.py

```diff
@@ -334,6 +335,13 @@
 # django-cors-headers - https://github.com/adamchainz/django-cors-headers#setup
 CORS_URLS_REGEX = r"^/api/.*$"
 
+# By Default swagger ui is available only to admin user. You can change permission classs to change that
+SPECTACULAR_SETTINGS = {
+    "TITLE": "{{ cookiecutter.project_slug }} API",
+    "DESCRIPTION": "Lets rock and then roll!",
```

Comment:
Maybe "API documentation for {{ cookiecutter.project_name}}".

Reply:
Fixed this